comfy_cli/command/distribution.py

```
@@ -1184,6 +1511,59 @@ def update_cmd(
+        renderer.success(f"Uploaded {file_path.name} as {blob_id}")
+        renderer.info(f'reference it in a definition as "blobId": "{blob_id}", then `comfy distribution update`')
+    renderer.emit(
+        {"blobId": blob_id, "kind": kind, "filename": file_path.name, "sha256": sha256, "sizeBytes": size_bytes},
+        command="distribution blob-upload",
+        changed=True,
+    )
+
+
+@app.command("blobs", help="List the workspace's uploaded private files.")
+@tracking.track_command("distribution")
+def blobs_cmd(
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)